src/dev/storage/ide_disk.hh

```diff
@@ -231,7 +231,7 @@ class IdeDisk : public SimObject
     /** Drive identification structure for this disk */
     struct ataparams driveID;
     /** Data buffer for transfers */
-    uint8_t *dataBuffer;
+    uint8_t *dataBuffer=nullptr;
```

Comment:
1. Needs spaces around the `=`.
2. Probably should be in the intializer list in the constructor since that's where all the other initializations are.